bin/cij_extractor

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+"""
+ CIJOE test extractor
+"""
+from __future__ import print_function
+import argparse
+import sys
+import os
+
+import cij.extractor
+import cij.runner
+import cij.util
+import cij
+
+
+def parse_args():
+    """Parse command-line arguments for cij_extractor"""
+
+    cij_evars = cij.paths_from_env("CIJ", ["TEMPLATES"])
+    for key, val in cij_evars.items():
+        if val is None:
+            cij.err(
+                f"rprtr:rprtr: failed parsing environment variable 'CIJ_{key}'"
+            )
+            return None
+
+    # Parse the Command-Line
+    prsr = argparse.ArgumentParser(
+        description='cij_extractor - CIJOE Test data extractor',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    prsr.add_argument(
+        'output',
+        help="Path to test output",
+        default=os.getcwd()
+    )
+    prsr.register('action', 'custom_extend', cij.util.ExtendAction)
+    prsr.add_argument(
+        "--extractor",
+        help="Name of, or path to, one or more extractor(s)",
+        action="custom_extend",
+        choices=[e.name for e in cij.extractor.find_extractors()],
```

Comment:
Great!

Reply:
Thanks! I randomly stumbled upon it and found that it fit very well. It even lists the existing ones if you don't supply one